README.md

```diff
@@ -3,3 +3,9 @@ The metapackage for robotont
 
 [![Build Status](https://travis-ci.org/robotont/robotont.svg?branch=master)](https://travis-ci.org/robotont/robotont)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+**Cloning the repository:**
+
+```bash
+git clone --recurse-submodules https://github.com/robotont/robotont.git
```

Comment:
I would use - - recursive, which is shorter @kruusamae what do you think?

Reply:
nõus, ise olen ka alati `--recursive` kasutanud